### PR TITLE
update release-1.23 config.toml for release 1.25

### PR DESCRIPTION
**Pull request by cathchu**
update release-1.23 config.toml for new release 1.25

Version were updated according to [latest patch releases](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md)

/cc @kcmartin @didicodes @sethmccombs @krol3 @reylejano

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview:
- https://deploy-preview-36178--k8s-v1-23.netlify.app/releases/
- https://deploy-preview-36178--k8s-v1-23.netlify.app/releases/version-skew-policy/

File: config.toml [hunk 1]
```diff
@@ -139,11 +139,11 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
 showedit = true
 
-latest = "v1.24"
+latest = "v1.25"
 
-fullversion = "v1.23.6"
+fullversion = "v1.23.10"
 version = "v1.23"
-githubbranch = "main"
+githubbranch = "v1.23.10"
```
Comment (sftim) [inline config.toml:146]: I think this stays as `main`, actually.

Comment (sftim) [inline config.toml:146]: This doesn't look right.

File: config.toml [hunk 2]
```diff
@@ -179,40 +179,40 @@ js = [
...
+githubbranch = "v1.25.0"
+docsbranch = "main"
+url = "https://kubernetes.io"
+
+[[params.versions]]
+fullversion = "v1.24.4"
 version = "v1.24"
-githubbranch = "v1.24.0"
+githubbranch = "v1.24.4"
 docsbranch = "main"
```
Comment (sftim) [inline config.toml:192]:
```suggestion
docsbranch = "release-1.24"
```

File: config.toml [hunk 3]
```diff
@@ -139,11 +139,11 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
 
-latest = "v1.24"
+latest = "v1.25"
 
-fullversion = "v1.23.6"
+fullversion = "v1.23.10"
 version = "v1.23"
-githubbranch = "main"
+githubbranch = "v1.23.10"
 docsbranch = "main"
```
Comment (sftim) [inline config.toml:147]: Should this change from `main`?

Comment (reylejano) [reply config.toml:147]: yes it should be `release-1.23`

**Comment by reylejano** [comment]
closing this pr
opened https://github.com/kubernetes/website/pull/36200 to address comments
/close

**Pull request closed without merging**